cluster/charts/crossplane/templates/deployment.yaml

```diff
@@ -37,6 +41,9 @@ spec:
         name: {{ .Chart.Name }}
         resources:
           {{- toYaml .Values.resourcesCrossplane | nindent 12 }}
+        ports:
+        - name: http
+          containerPort: 8080
```

Comment:
should we make this able to be removed if desired?

Reply:
Technically that's possible. Maybe add a new set(s) of variable to chart to control this. And also if this gets removed the `prometheus.io/` related annotations should be removed too.